### DISCUSSION TITLE
[Feature request] Date + Kebab Filenames

### DISCUSSION THREAD

**Discussion by benyanke** [GitJournal/GitJournal#870] [Feature Requests]
It would be very useful to be able to do date + note title based file names.

For example:

`yyyy-mm-dd_[kebab-case-title].[extension]`

This is how I typically format my notes by hand in my own notes repo, and it would be useful for GitJournal to be able to do the same! It allows easier interaction with notes in the repo without GitJournal, for example, on a PC with the checked out repo.

**Comment by justinsilvestre** [comment]
This is exactly what I have been doing in my own notes repo.

This lets me search easily through notes **by date AND title** in a text editor like Visual Studio Code.
It's nice to be able to access my notes on any computer with git + a good text editor.

There have been other requests for features which would make this possible, like #573 #204.
